### PR TITLE
Moved themes menu to view menu  #180

### PR DESCRIPTION
**Pull request by SlowLogicBoy**
#180

### CODE REVIEW

File: dnSpy/dnSpy/Themes/ThemesMenu.cs [hunk 1]
```diff
@@ -20,11 +20,18 @@
...
 using System.ComponentModel.Composition;
+using System.Diagnostics;
 using dnSpy.Contracts.Menus;
 using dnSpy.Contracts.Themes;
 using dnSpy.Properties;
 
 namespace dnSpy.Themes {
+	[ExportMenuItem(OwnerGuid = MenuConstants.APP_MENU_VIEW_GUID, Guid = MenuConstants.APP_MENU_THEMES_GUID, Order = MenuConstants.ORDER_APP_MENU_THEMES, Header = "res:Menu_Themes", Group = MenuConstants.GROUP_APP_MENU_VIEW_OPTS)]
+	sealed class ThemesMenuItem : MenuItemBase
+	{
```
Comment (0xd4d) [inline dnSpy/dnSpy/Themes/ThemesMenu.cs]: Should be on the previous line

Comment (SlowLogicBoy) [inline dnSpy/dnSpy/Themes/ThemesMenu.cs]: You mean in the dnSpy/dnSpy/Menus/AppMenus.cs ?

**Comment by 0xd4d** [comment]
Curly braces aren't on new lines. See the rest of the code.

File: dnSpy/dnSpy/Themes/ThemesMenu.cs (hunk 1 above)
Comment (0xd4d) [inline dnSpy/dnSpy/Themes/ThemesMenu.cs]: Curly braces aren't on new lines. See the rest of the code.

**Comment by SlowLogicBoy** [comment]
Sorry forgot to fix that, tabs already gave me a hard time :)

**Comment by 0xd4d** [comment]
Thanks!

**Pull request merged**